templates/receipt/pdf_receipt.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * This example showcases the usage of creator templates to create a receipt document
+ *
+ * Run as: go run pdf_receipt.go
+ */
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"log"
+	"os"
+
+	"github.com/unidoc/unipdf/v3/common"
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/creator"
+)
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io.
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+	common.SetLogger(common.NewConsoleLogger(common.LogLevelDebug))
+}
+
+// Field represents field item.
+type Field struct {
+	FieldName  string `json:"FieldName"`
+	FieldValue string `json:"FieldValue"`
+}
+
+// Receipt represent Receipt object.
+type Receipt struct {
+	Title  string
+	Fields []Field
+}
+
+func main() {
+	c := creator.New()
+	c.SetPageMargins(15, 15, 20, 20)
+	c.SetPageSize(creator.PageSizeA5)
+	tpl, err := readTemplate("./templates/main.tpl")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	filePath := "./contents/receipt.json"
```

Comment:
You don't use this variable anywhere else. You can just pass the value to `readReceipt`:
```go
receipt, err := readReceipt("contents/receipt.json")
```